app/Http/Controllers/ArticleController.php

```diff
@@ -4,12 +4,21 @@
 
 use Illuminate\Http\Request;
 use App\Models\Article;
+use Psr\Log\LoggerInterface;
 
 class ArticleController extends Controller
 {
-    public function show($articleId){
-        
+    public function show($articleId, Request $request, LoggerInterface $logger){
```

Comment:
This code is incomplete and was finished at lesson.

Reply:
I added missing part and complete this code

Reply:
Not valid changes, re-watch the third lesson.

Reply:
Changed